_i18n/ar/resources/moneropedia/sidechain.md

```diff
@@ -0,0 +1,17 @@
+---
+entry: "sidechain"
+terms: ["sidechain", "side-chain", "auxillary-chain"]
+summary: "a blockchain that is merge mined with a primary blockchain"
+---
+
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+### The Basics
+
+A sidechain (also known as an auxillary chain) is a @blockchain that is @merge-mined with Monero or other cryptocurrency. In most cases the sidechain will have a lower difficulty than the primary blockchain.
+
+A pegged sidechain is a sidechain that allows you to transfer assets between different blockchains by pegging the value of one asset to another on an alternate blockchain. Currently there are no pegged sidechains that exist for Monero.
```

Comment:
I would rephrase this or people won't understand why a pegged blockchain is mentioned. Something like

```suggestion
A sidechain that that allows you to transfer assets between different blockchains is called a "pegged blockchain". It works by pegging the value of one asset to another on an alternate blockchain. Currently there are no pegged sidechains that exist for Monero.
```

I would also add between brackets a short explanation of what "pegging" means.

Reply:
> a short explanation of what "pegging" means.

Ack. We don't want people having to search the internet for that.